src/templates/settings.twig

```diff
@@ -28,6 +28,22 @@
     suggestEnvVars: true
 }) }}
 
+{% if cdn_determined_from_env %}
+    {% if cdn_enabled_from_env %}
+        <input type="checkbox" name="enableCDNCacheInvalidation" value="1" checked disabled>
+        <label style="color: grey" for="enableCDNCacheInvalidation">Enable Google Cloud CDN Cache Invalidation on Save (currently set by env)</label>
+    {% else %}
+        <input type="checkbox" name="enableCDNCacheInvalidation" value="0" disabled>
+        <label style="color: grey" for="enableCDNCacheInvalidation">Enable Google Cloud CDN Cache Invalidation on Save (currently set by env)</label>
+    {% endif %}
+{% else %}
```

Comment:
Go ahead and remove this `else`, we want to remove the option entirely to select it in the settings, maybe later we'll either expose it or remove the disabled checkbox altogether if it doesn't prove useful